static/docs/commands-reference/repro.md

```diff
@@ -67,3 +125,15 @@ Reproduce the part of the pipeline where `Posts.tsv.dvc` is the target DVC file:
     Reproducing 'Posts.tsv.dvc':
             python code/xml_to_tsv.py data/Posts.xml data/Posts.tsv python
 ```
+
+Reproduce a stage located in a subdirectory:
+
+```dvc
+    $ dvc repro --force --single-item --cwd data Posts.xml.zip.dvc 
+    Stage 'Posts.xml.zip.dvc' didn't change.
```

Comment:
put an empty line like in the examples above, or remove it everywhere

Reply:
Yes